hs_core/views/__init__.py

```diff
@@ -719,30 +758,40 @@ def create_resource(request, *args, **kwargs):
     res_title = request.POST['title']
 
     resource_files = request.FILES.getlist('files')
-
+    fed_res_file_names=[]
     irods_fnames = request.POST.get('irods_file_names')
+    federated = True if request.POST.get("irods_federated").lower()=='true' else False
+    do_copy = request.POST.get("copy-or-move")
+    fed_copy = True
+    if do_copy:
```

Comment:
@pkdash You are right, this ```if``` is not needed here since that ```copy-or-move``` input field always has a value with default value being "copy" - the field is just hidden for the cases where no file is selected from a federated zone for resource creation request. I will remove the unneeded ```if``` statement then.